package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "extract-text-webpack-plugin",
+  "name": "extract-text-webpack-plugin-steamer",
```

Comment:
Should stay `extract-text-webpack-plugin` in PR unless your not to planning to release your fork